Rubberduck.RibbonDispatcher/Sample Ribbons/SampleRibbon/src/ThisWorkbook.cls

```diff
@@ -0,0 +1,44 @@
+VERSION 1.0 CLASS
```

Comment:
File extension is inconsistent with Rubberduck's export functionality; these will import as regular class modules - use `.doccls` for document-module classes.

Reply:
On the To-Do list - haven't caught up yet.